worker/codalabworker/worker.py

```diff
@@ -105,9 +105,7 @@ def _checkin(self):
             elif action_type == 'read':
                 self._read(socket_id, response['uuid'], response['path'], response['read_args'])
             elif action_type == 'netcat':
-                self._netcat(
-                    socket_id, response['uuid'], response['port'], response['message'].encode()
-                )
+                self._netcat(socket_id, response['uuid'], response['port'], response['message'])
```

Comment:
I think this change might be causing test failures -- perhaps _netcat requires the message to be bytes?

Reply:
Yeah, something's going on.  I think our convention should be that `_netcat` and `_write` take strings (for consistency).